ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -79,6 +79,10 @@ tests:
   commands: make name-check
   container:
     from: src
+- as: code-check
```

Comment:
@Xia-Zhao-rh add `optional: true` firstly. after it is stable, we make new PR to make it required.